npm-search/assets/js/app.js

```diff
@@ -168,7 +168,15 @@ var Plugin = React.createClass({
             </li>
         )
     }
-})
+});
+
+var timer=0;
```

Comment:
I see - Can timer be a member of `PluginList` class?  Also, you might want to initialize to null instead of 0.